### PR TITLE
Update memoryReservation for consul agents

### PR DESCRIPTION
**Pull request by tfhartmann**
This PR updates some of the defaults for the consul/registrator agents. In particular it sets the default mem reservation to lower so that consul and registrator don't dominate smaller instances

### CODE REVIEW

**Review by hakamadare — APPROVED** [review]
that all looks good. i might have picked 16 or 32 MB rather than 20 for a default, but that’s perhaps just me being fussy :)

File: .circleci/config.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@ jobs:
           command: if [ `terraform fmt | wc -c` -ne 0 ]; then echo "Some terraform files need be formatted, run 'terraform fmt' to fix"; exit 1; fi
       - run:
           name: "get tflint"
-          command: apk add wget ; wget https://github.com/wata727/tflint/releases/download/v0.4.2/tflint_linux_amd64.zip ; unzip tflint_linux_amd64.zip
+          command: apk update && apk add wget ; wget https://github.com/wata727/tflint/releases/download/v0.5.4/tflint_linux_amd64.zip ; unzip tflint_linux_amd64.zip
```
Comment (antonbabenko) [inline .circleci/config.yml:18]: Hi @tfhartmann @hakamadare 👋  Good to *see* you again!

Hope you don't mind to use latest :)

Comment (tfhartmann) [reply .circleci/config.yml:18]: Don't mind at all!

**Comment by tfhartmann** [comment]
@hakamadare 32 it is!

**Pull request merged**